Runtime/Scripts/BaseMeshSync.cs

```diff
@@ -1330,9 +1352,32 @@ private EntityRecord UpdateMeshEntity(MeshData data, MeshSyncPlayerConfig config
         public static Action ProBuilderBeforeRebuild;
         public static Action ProBuilderAfterRebuild;
 #endif
+      
+//--------------------------------------------------------------------------------------------------------------------------------------------------------------
+        
+#if AT_USE_HDRP
 
+        void UpdatePathTracingState() {
+            m_pathTracingExists = false;
+            if (!HDRPUtility.IsRayTracingActive()) 
+                return;
+            
+            SceneComponents<Volume> sceneVolumes = SceneComponents<Volume>.GetInstance();
+            sceneVolumes.Update();
+            m_pathTracingExists = HDRPUtility.IsPathTracingActive(sceneVolumes.GetCachedComponents());
```

Comment:
What exactly does this code do? seems like its updating scene volumes per frame. How is this related to MeshSync?

Reply:
`Update()` here means updating cached components, so that we only do `Object.FindObjectsOfType` per frame

Reply:
Why do we cache and update these components?

Reply:
I suppose this should be implemented by the user of the package, but we want to allow users to get PathTracing working when using MeshSync without any effort from their side?